wasm_runtime/src/config.rs

```diff
@@ -31,6 +31,11 @@ impl WasmConfig {
     /// Returns Result<(), String>, so that in case of error the String will contain the reason.
     /// 
     pub fn create(config_id: &str) -> Result<(), String> {
+        // safety check for empty id
+        if config_id.len() == 0 {
+            let error_msg = format!("ERROR! Can't create WasmConfig for an empty config_id!");
+            return Err(error_msg);
```

Comment:
You can avoid using `format!` in this case since the `error_msg` doesn't interpolate any variable:

```suggestion
            return Err("ERROR! Can't create WasmConfig for an empty config_id!");
```

Reply:
Given create returns a `Result<(), String>` you might need to convert it to String, `return Err("ERROR! Can't create WasmConfig for an empty config_id!".to_string());`. In this case, the `format!` macro was doing this automatically behind the scenes.

Reply:
Oh! I missed the `Result` signature  🤦. Great catch @ereslibre! Let me update my comment